src/olympia/reviewers/utils.py

```diff
@@ -1659,7 +1661,7 @@ def unreject_latest_version(self):
 
         self.set_file(amo.STATUS_AWAITING_REVIEW, self.version.file)
         self.log_action(amo.LOG.UNREJECT_VERSION)
-        self.addon.update_status(self.user)
+        self.addon.update_status()
```

Comment:
this (and below) were drive-by fixes when I was looking into update_status - the arg is supposed to be a `Version` instance to ignore, so passing `self.user` was always useless.